### ISSUE TITLE
[EEZ Studio / LVGL] Prevent the use of internal / reserved variables

### ISSUE THREAD

**Issue by jcuypers** [eez-open/studio#439]
When defining variables in a project there should be some basic whether or not the variables are already used somewhere else in the project.

**Comment by mvladic** [comment]
I'm not sure I understand what you're trying to say. Can you elaborate on this a bit?

**Comment by jcuypers** [comment]
> I'm not sure I understand what you're trying to say. Can you elaborate on this a bit?

Yes sure, so if would choose/create a variable named "status" in my project, all works fine in the editor.  After generating the actual LVGL code, there are a lot of compiler errors since somewhere else in the framework this variable name is already used.

I know its a corner case and probably a lot of work to avoid, but its good to be known at least.
